app/controllers/application_controller.rb

```diff
@@ -25,7 +25,7 @@ def after_sign_in_path_for(resource_or_scope)
   end
 
   def guest_or_customer_id
-    customer_signed_in? ? current_customer.id : session['session_id']
+    customer_signed_in? ? current_customer.id : session['session_id'].gsub(/\D/, '').to_i / 10000000000
```

Comment:
Ugh, this isn't ideal...but I noticed the session id was not writing to the db because of a type error. As a hack, I remove non-numerical characters and then divide by a billion to end up with a number that fits in the pg database. Let's figure out a better way to do this.